example/mm_jobman.sh

```diff
@@ -157,8 +158,11 @@ create_upload_commands() {
         local source="${upload_local[$i]}"
         local destination="${upload_remote[$i]}"
 
+        # Add mkdir command
+        cmd+="mkdir -p '$source' && "
```

Comment:
for upload  it does not help to mkdir on source. Instead it should check and make sure that source folder exists -- please implement it that way (2nd time i raise this)

Reply:
Sure, I can check if the source directory exists and only make it if it does not
```
        # Add mkdir command for source folder if dne
        if [ ! -d "$source" ]; then
          cmd+="mkdir -p '$source' && "
        fi
 ```